optimum/pipelines.py

```diff
@@ -0,0 +1,104 @@
+from sre_constants import ANY
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
+
+from transformers import (
+    AutoTokenizer,
+    FeatureExtractionPipeline,
+    Pipeline,
+    PreTrainedTokenizer,
+    QuestionAnsweringPipeline,
+    TextClassificationPipeline,
+    TokenClassificationPipeline,
+    ZeroShotClassificationPipeline,
+)
+from transformers import pipeline as transformers_pipeline
+from transformers.feature_extraction_utils import PreTrainedFeatureExtractor
+from transformers.pipelines import TASK_ALIASES
+
+from optimum.utils import is_onnxruntime_available
+
+
+SUPPORTED_TASKS = {}
+
+if is_onnxruntime_available():
+    from optimum.onnxruntime import (
+        ORTModel,
+        ORTModelForFeatureExtraction,
+        ORTModelForQuestionAnswering,
+        ORTModelForSequenceClassification,
+        ORTModelForTokenClassification,
+    )
+
+    SUPPORTED_TASKS = {
```

Comment:
Should we add `text-generation` to this list?

Reply:
Done